src/Viewer.tsx

```diff
@@ -449,8 +450,19 @@ function Viewer(): ReactElement {
           datasetKey = newCollection.getDefaultDatasetKey();
         } else {
           // Try loading the collection, with the default collection as a fallback.
-          newCollection = await Collection.loadCollection(collectionUrlParam || DEFAULT_COLLECTION_PATH);
-          datasetKey = datasetParam || newCollection.getDefaultDatasetKey();
+          try {
+            newCollection = await Collection.loadCollection(collectionUrlParam || DEFAULT_COLLECTION_PATH);
+            datasetKey = datasetParam || newCollection.getDefaultDatasetKey();
+          } catch (error) {
+            console.error(error);
+            notificationApi["error"]({
+              message: "Error loading dataset:",
```

Comment:
does the url of the dataset come in here via the `error`?  Would it be good to add `collectionUrlParam` to the message explicitly here?